### ISSUE TITLE
Applying the limit on one product only

### ISSUE THREAD

**Issue by ValeryKuz** [nexcess/limit-orders#66]
is there any option to set orders limit for specific product only?

**Comment by stevegrunwell** [comment]
Hi @ValeryKuz,

[This has come up before](https://wordpress.org/support/topic/can-this-applied-to-single-products/), and it's something of a different paradigm: instead of counting down the number of orders to accept before turning off checkout (which this plugin does), limiting orders based on a single product would likely be best-accomplished by an automatically-renewing inventory.

For example, imagine you run a bakery: every day, you bake five dozen bagels, so every morning the inventory should be 60 (5*12). Once you're out of bagels, you no longer want people to be able to buy them (as they're out of stock) _but_ you don't want to prevent them from purchasing other baked goods (donuts, muffins, etc.).

Limit Orders, meanwhile, is designed to turn off the entire checkout process once a certain number of orders has been reached. Going back to the bakery example, it would be perfect for a bakery that says "we have limited staff and can only prepare 100 online orders in a day, regardless of what delicious treats are in the orders".

A better approach for your use case would likely be a plugin/script to automatically reset the inventory on a particular product on a schedule (e.g. "refresh the bagel inventory every day at 7am"). We've discussed building such a plugin, but sadly it isn't something that we've gotten around to yet.
